### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by mt-rory**
add detail about how to update the web-font and test on local.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,14 +6,29 @@ You have Node installed at v10.0.0+ and Yarn at v1.2.0+.
...
+    - replace the folder `/fonts` with unzip folder `/fonts`
+    - replace the file `/css/style.css` with unzip file `/style.css`
+    - replace the file `/selection.json` with unzip file `/selection.json`
+
+### Test the new icon it's work on local.
+1. Under the mt-web-icon run
+    - `yarn`
+    - `yarn build`
+2. Under your project, install the local mt-web-icon package
+   - `rm -rvf ./node_modules/mt-web-icons`
```
Comment (mt-neus) [inline README.md]: you don't need the v (verbose). just with `rm -rf ./node_modules/mt-web-icons` is enough

Comment (mt-roberto) [reply README.md]: I think it works both ways, anyway. ;)

Comment (mt-roberto) [inline README.md]:
```suggestion
   - `rm -rf ./node_modules/mt-web-icons`
```

Comment (mt-neus) [reply README.md]: it does, but for the readme you don't need to be verbose ;)

**Pull request merged**